test/functional/apps/management/_field_formatter.ts

```diff
@@ -32,11 +32,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         'test_field_formatters',
         'test_logstash_reader',
       ]);
-      await esArchiver.load('test/functional/fixtures/es_archiver/discover');
+      await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover');
       await kibanaServer.uiSettings.replace({});
     });
 
     after(async function afterAll() {
+      await kibanaServer.importExport.unload('test/functional/fixtures/kbn_archiver/discover');
       await PageObjects.settings.navigateTo();
       await esArchiver.emptyKibanaIndex();
```

Comment:
Is `await esArchiver.emptyKibanaIndex();` just a wrapper for `esArchiver.load('emptyKibana')` ?  
Or an `esArchiver.unload()`?  
If so, I don't think it should be necessary.  And we really want to avoid using the `esArchiver.unload()` on the .kibana index.

Reply:
Not totally sure about the differences, but `esArchiver.load('emptyKibana')` loads this obj:
```
{
  "type": "doc",
  "value": {
    "id": "config:6.0.0-alpha1",
    "index": ".kibana",
    "source": {
      "config": {
        "buildNum": 8467,
        "dateFormat:tz": "UTC"
      },
      "type": "config"
    }
  }
}
```
and `await esArchiver.emptyKibanaIndex()` looks like this:
```
export async function emptyKibanaIndexAction({
  client,
  log,
  kbnClient,
}: {
  client: KibanaClient;
  log: ToolingLog;
  kbnClient: KbnClient;
}) {
  const stats = createStats('emptyKibanaIndex', log);

  await cleanKibanaIndices({ client, stats, log });
  await migrateKibanaIndex(kbnClient);
  stats.createdIndex('.kibana');
  return stats.toJSON();
}
```
Looks like the latter is more involved.